### PR TITLE
Add get_vlan_interfaces method on dell switches. 

### PR DESCRIPTION
**Pull request by emmurd**

### CODE REVIEW

File: tests/adapters/switches/dell10g_test.py [hunk 1]
```diff
@@ -293,6 +293,26 @@ def test_get_vlan_with_unknown_number(self):
 
         assert_that(str(expect.exception), equal_to("Vlan None not found"))
 
+    def test_get_vlan_interfaces(self):
+        self.mocked_ssh_client.should_receive("do").with_args("show vlan id 1000", wait_for=("--More-- or (q)uit", "#"), include_last_line=True).once().ordered().and_return([
+            "VLAN       Name                         Ports          Type      Authorization",
+            "-----  ---------------                  -------------  -----     -------------",
+            "1000   MyVlanName                       ch2-3,ch5-6,   Static    Required",
```
Comment (JoProvost) [inline tests/adapters/switches/dell10g_test.py]: My I suggest that we test with "ch2-3,ch5-7" and assert that we get 'port-channel 2', 'port-channel 3', 'port-channel 5', 'port-channel 6', 'port-channel 7' ?

Just to make sure...

**Comment by emmurd** [comment]
Test with ch5-7 added

**Comment by JoProvost** [comment]
:+1:

File: tests/adapters/switches/dell_test.py [hunk 2]
```diff
@@ -398,9 +398,50 @@ def test_get_vlan_with_interface_multipage(self):
...
+            "                                        1/xg4,2/g2,"
+        ])
+
+        vlan_interfaces = self.switch.get_vlan_interfaces(1000)
+
+        assert_that(vlan_interfaces, equal_to(['port-channel 2', 'port-channel 3', 'port-channel 5', 'port-channel 6', 'port-channel 7', 'ethernet 1/g4', 'ethernet 1/g7',
                                               'ethernet 1/g13', 'ethernet 1/g17', 'ethernet 1/g18', 'ethernet 1/g19', 'ethernet 1/g20', 'ethernet 1/xg4',
                                                'ethernet 2/g2']))
 
+    def test_get_vlan_interfaces_with_unknown_vlan(self):
```
Comment (idjaw) [inline tests/adapters/switches/dell_test.py]: test name should specify what should happen. 'test_get_vlan_interfaces_with_unknown_vlan_raises'

File: tests/adapters/switches/dell10g_test.py [hunk 3]
```diff
@@ -293,6 +293,26 @@ def test_get_vlan_with_unknown_number(self):
...
+            "-----  ---------------                  -------------  -----     -------------",
+            "1000   MyVlanName                       ch2-3,ch5-7,   Static    Required",
+            "                                        1/g4,1/g7,"
+        ])
+
+        vlan_interfaces = self.switch.get_vlan_interfaces(1000)
+
+        assert_that(vlan_interfaces, equal_to(['port-channel 2', 'port-channel 3', 'port-channel 5', 'port-channel 6', 'port-channel 7', 'ethernet 1/g4', 'ethernet 1/g7']))
+
+    def test_get_vlan_interfaces_with_unknown_vlan(self):
```
Comment (idjaw) [inline tests/adapters/switches/dell10g_test.py]: change to: test_get_vlan_interfaces_with_unknown_vlan_raises

**Comment by emmurd** [comment]
@idjaw DONE!!

**Pull request merged**